tests/end_to_end/helpers.h

```diff
@@ -86,17 +86,15 @@ typedef struct _ebpf_free_memory
 
 typedef std::unique_ptr<uint8_t, ebpf_free_memory_t> ebpf_memory_t;
 
+int
+bpf_link__destroy(bpf_link* link);
```

Comment:
Why is this prototype needed?  It's already in include\bpf\libbpf.h.  Can't that be included?

Reply:
Add // comments explaining why if there's no easy workaround.

Reply:
Unresolving since there are still no // comments explaining why this prototype isn't in an included header file.